src/app/pages/ParatimeDashboardPage/ParaTimeSnapshot.tsx

```diff
@@ -53,7 +53,7 @@ export const ParaTimeSnapshot: FC<{ scope: SearchScope }> = ({ scope }) => {
           <Nodes scope={scope} />
         </StyledGrid>
         <StyledGrid item xs={22} md={5}>
-          {!mainToken.free && <TokenPriceCard token={mainToken} />}
+          {showFiatValues && !mainToken.free && <TokenPriceCard token={mainToken} />}
```

Comment:
what non-free token are we hiding now?

Reply:
The Pontus-X guys don't want to see _any_ fiat values anywhere in their version, not even the price card. So, in their deployment, `showFiatValue` will be set to `false` via `.env`, and so the token price card (for EUROe) will be hidden.